site/content/en/docs/tasks/run/wrapped_custom_jobs.md

```diff
@@ -0,0 +1,58 @@
+---
+title: "Run A Wrapped Custom Job"
+linkTitle: "Custom Job"
+date: 2025-01-14
+weight: 7
+description: >
+  Use an AppWrapper to Run a Custom Job on Kueue.
+---
+
+This page shows how to use [AppWrappers](https://project-codeflare.github.io/appwrapper/) to make
+Kueue's scheduling and resource management capabilities available to Job types that do not have a dedicated
```

Comment:
```suggestion
Kueue's scheduling and resource management capabilities available to Workload types that do not have a dedicated
```
nit: I'm also ok to keep as is for consistency - as you prefer, but I think we could start to write new docs emphasizing Kueue is not just for Jobs, but generic Workloads (jobs and serving). Especially this seems important in the context of AppWrapper which was designed to support arbitrary workloads. 

Feel free to keep as is, or revise the content with that remark in mind.

Reply:
Good suggestion.  Done